### PR TITLE
update notebooks with automatic download of datasets

### PR DESCRIPTION
**Pull request by gonlairo**
PR to update notebooks with automatic download of datasets

### CODE REVIEW

**Review by stes — CHANGES_REQUESTED** [review]
Let's add the notebooks again with only the small change at the beginning of the header

File: Demo_cohomology.ipynb [hunk 1]
```diff
@@ -88,11 +136,19 @@
...
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "achilles.jl\n"
+     ]
+    }
+   ],
    "source": [
-    "hippocampus_pos = cebra.datasets.init('rat-hippocampus-single-achilles')"
+    "ls data/rat_hippocampus"
```
Comment (MMathisLab) [inline Demo_cohomology.ipynb:151]: this seems hard-coded for unix; what about windows support? @gonlairo

File: Demo_hippocampus_multisession.ipynb [hunk 2]
```diff
@@ -437,9 +480,9 @@
  ],
  "metadata": {
```
Comment (stes) [inline Demo_hippocampus_multisession.ipynb:481]: I think here we want to keep the original kernel name in the metadata?

File: Demo_primate_reaching.ipynb
Comment (stes) [inline Demo_primate_reaching.ipynb:1]: this here seems like a pretty big change, intended?

File: Demo_primate_reaching_mse_loss.ipynb
Comment (stes) [inline Demo_primate_reaching_mse_loss.ipynb:1]: this here seems like a pretty big change, intended?

File: Demo_synthetic_exp.ipynb
Comment (stes) [inline Demo_synthetic_exp.ipynb:1]: this here seems like a pretty big change, intended?

File: README.rst [hunk 3]
```diff
@@ -47,6 +47,11 @@ download them and check the loading directory is correct in the
...
 ``./data/`` directory in the repository root.
 
+For the **hippocampus dataset** the data is automatically downloaded when you call 
+``cebra.datasets.init()``. The default path is the ``./data`` folder
+in the repository root. It will be possible to automatically download the remaining
+datasets soon.
+
 For different paths, you can specify the ``CEBRA_DATADIR=...``
 environment variable. You can do this by placing
 ``import os; os.environ['CEBRA_DATADIR'] = "path/to/your/data"`` at the
```
Comment (stes) [inline README.rst:57]: Can we add the code snippet from @MMathisLab diff here (#3 ) to the README here as well? We can remove it again once all datasets support auto-download.

**Review by stes — CHANGES_REQUESTED** [review]
some smaller comments

**Review by MMathisLab — APPROVED** [review]

**Review by stes — APPROVED** [review]

**Pull request merged**